libtsuba/src/RDGPartHeader.cpp

```diff
@@ -97,11 +97,8 @@ RDGPartHeader::MakeJson(const katana::Uri& partition_path) {
 
 katana::Result<RDGPartHeader>
 RDGPartHeader::Make(const katana::Uri& partition_path) {
-  katana::Result<RDGPartHeader> res = MakeJson(partition_path);
-  if (!res) {
-    return res.error();
-  }
-  return res;
+  auto part_header = KATANA_CHECKED(MakeJson(partition_path));
+  return part_header;
```

Comment:
actually this could just be:
`return MakeJson(partition_path);`
or if you still want the call trace
`return KATANA_CHECKED(MakeJson(partition_path));`

Reply:
Ideally I refactor the rest of the class to use KATANA_CHECKED, but that'll be for another PR